pkg/providers/anthropic/chat_stream.go

```diff
@@ -2,15 +2,102 @@ package anthropic
 
 import (
 	"context"
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
 
 	"glide/pkg/api/schemas"
 	"glide/pkg/providers/clients"
 )
 
+type Client struct {
+	APIKey string
+}
+
 func (c *Client) SupportChatStream() bool {
-	return false
+	return true
+}
+
+func (c *Client) ChatStream(ctx context.Context, chatReq *schemas.ChatRequest) (clients.ChatStream, error) {
```

Comment:
Over the weekend, I was finalizing the ChatStream interface and added this `Open()` method where the request initialization is supposed to happen ([here is an example from OpenAI](https://github.com/EinStack/glide/blob/develop/pkg/providers/openai/chat_stream.go#L41-L55)). Would good to move most of this code to `AnthropicChatStream.Open()` so we can properly track the initial request latency.